uniswap_example.py

```diff
@@ -10,8 +16,33 @@
     parser.add_argument(
         "--n_steps", type=int, default=100, help="Number of steps of the simulation"
     )
+    parser.add_argument(
+        "--batch_runner",
+        action="store_true",
+        help="Run batch of simulations over different simulation parameters",
+    )
     args = parser.parse_args()
 
+    # run a single simulation
     results = simulations.uniswap.sim.run_from_cache(args.seed, args.n_steps)
-
     simulations.uniswap.plotting.plot_results(results)
+
+    # run a batch of simulations
+    if args.batch_runner:
+        parameters_samples = [
+            dict(mu=mu, sigma=sigma)
+            for mu, sigma in product([0.0, 0.1, -0.1], [0.1, 0.2, 0.3])
+        ]
+
+        with open(os.path.join("simulations", "uniswap", "cache.json"), "r") as f:
+            cache_json = json.load(f)
+        cache = verbs.utils.cache_from_json(cache_json)
```

Comment:
Do we not have a cache from file function? If not we should add

Reply:
I don't think so. Will add in new branch of verbs